lib/infra/quests/couch/db.ex

```diff
@@ -29,7 +30,14 @@ defmodule Infra.Quests.Couch.Db do
 
   @impl PointQuest.Behaviour.Quests.Repo
   def get_quest_by_id(quest_id) do
-    case Projectionist.Store.get(Infra.Quests.Couch.Store, quest_id) do
+    # TODO: Add snapshots
+    init_quest = Quest.init()
```

Comment:
Over the decision that I made in projectionist that the init func returns a tupple. Refatored that away to just be the init state of the model